### PR TITLE
#164658918: Fix issue with the social login

### PR DESCRIPTION
**Pull request by Dubby20**
#### What does this PR do?
- It creates a jwt token when a user login/register with their social account.

#### Description of Task to be completed?
- Creates a jwt token so that users can be authenticated.
- Users to receive a jwt token when they login/register with their social account

#### How should this be manually tested?
- Run `npm run start` to start the application.
- Make a `GET` request to `/api/v1/auth/twitter` which will generate a token when a user logs in or registers via social account.

### What are the relevant pivotal tracker stories?
- [#164658918](https://www.pivotaltracker.com/story/show/164658918)

#### Any background context you want to provide?
- None

#### Screenshots (if appropriate)
- None

#### Questions:
- N/A

### CODE REVIEW

**Review by codinger41 — APPROVED** [review]

**Comment by codinger41** [comment]
LGTM.

**Review by youngestdj — APPROVED** [review]
LGTM

File: index.js [hunk 1]
```diff
@@ -22,8 +25,27 @@ app.use(cors());
 app.use(morgan('combined'));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
-
+app.use(session({
```
Comment (9jaswag) [inline index.js:28]: @Dubby20 why are we using sessions again?

**Review by 9jaswag — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (Dubby20) [reply index.js:28]: @9jaswag we still have to use a session to be able to implement twitter login

Comment (9jaswag) [reply index.js:28]: @Dubby20 if we have to, lets skip Twitter login

Comment (9jaswag) [reply index.js:28]: @Dubby20 ??

Comment (Dubby20) [reply index.js:28]: Hmmm. Okay but I have to let the PO know

**Comment by 9jaswag** [comment]
Closing this in favour of PR https://github.com/andela/vidar-ah-backend/pull/50

**Pull request closed without merging**